Makefile

```diff
@@ -29,7 +29,7 @@ ifeq ($(processor),$(filter $(processor),aarch64 arm64))
 else ifeq ($(processor),$(filter $(processor),i386 x86_64))
     ARCH_CFLAGS = -maes -mpclmul -mssse3 -msse4.2
 else ifeq ($(processor),$(filter $(processor),arm armv7l))
-    ARCH_CFLAGS = -mfpu=neon
+    ARCH_CFLAGS = -mfpu=neon -marm
```

Comment:
We should avoid specifying the option `-marm` since it can be set internally in toolchain defaults.